git_pull/config.json

```diff
@@ -21,7 +21,7 @@
   "schema": {
     "deployment_key": ["str"],
     "deployment_key_protocol": "match(rsa|dsa|ecdsa|ed25519|rsa)",
-    "repository": "url",
+    "repository": "str",
```

Comment:
I think `match(^[\w]+://[\w\.\-\/]+$)` is better?

Reply:
If we want to get more specific, this will probably work better: `match((?:git|ssh|https?|git@[-\w.]+):(\/\/)?(.*?)(\.git)(\/?|\#[-\d\w._]+?))`

borrowed from https://github.com/jonschlinkert/is-git-url/blob/master/index.js